apollo-api/src/main/java/com/apollographql/apollo/api/internal/Consumer.java

```diff
@@ -0,0 +1,21 @@
+package com.apollographql.apollo.api.internal;
+
+/**
+ * Represents an operation that accepts a single input argument and returns no result. Unlike most other functional
+ * interfaces, {@code Consumer} is expected to operate via side-effects.
+ *
+ * <p>This is a <a href="package-summary.html">functional interface</a> whose functional method is {@link
+ * #accept(Object)}.
+ *
+ * @param <T> the type of the input to the operation
+ */
+public interface Consumer<T> {
```

Comment:
Did you use Consumer on purpose ? Why not call the class Mutator<T> or BuilderMutator<T> so as to avoid confusion with the other Consumers (reactivex, java.util, com.annimon.stream, maybe others ?)

Reply:
Just copied from Java8, but yeah we can rename it